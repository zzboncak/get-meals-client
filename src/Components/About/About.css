.parentView {
display: flex;
flex-direction: row;
flex-wrap: wrap;
}

.parentView p {
  align-content: space-between;

}

.avatar-image {
  height: 150px;
  width: 150px;
  border-radius: 50%;
  scale: 50%;
}

.item {
  flex: 1;
  flex: 1 0 0%;

}