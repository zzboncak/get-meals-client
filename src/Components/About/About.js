import React from 'react';
import './About.css';
import placeholder from './placeholder.jpg'
import VictoriaM from './VictoriaM.jpeg'
import ZackZ from './ZackZ.jpeg'
import PeggyS from './PeggyS.jpg'
import JohnL from './JohnL.jpeg'
import SejinH from './SejinH.jpeg'

class About extends React.Component {
  render() {
    return (

      <React.Fragment>

        <div className='about-header'>
          <h1>About Get Meals</h1>
          <p>Get Meals was created by a group of students at Thinkful who all of the like mind of wanting to create something that will help people. The app was born in the midst of the COVID-19 out of a want to help people easily find places where they can get free meals, the location of food banks and places that provide discounts to our military, healthcare workers, etc.</p>

          <h1>Meet the Team</h1>

        </div>

        <div className="parentView">

          <div className="item">
            <h2>John</h2>

            <img
              src={JohnL}
              alt="John Lee"
              className="avatar-image" />


            <p>John is a bioengineer transitioning to the software development world. He's just a nerd. When he has free time, John can be found doing something active outdoors (e.g. hiking a 14er, snowboarding). He is currently based out of Denver, CO. <a href="https://johlee92.github.io/personalPortfolio/" target="_blank" rel='noreferrer noopener'>Here</a> is his portfolio.</p>

          </div>



          <div className="item">
            <h2>Victoria</h2>

            <img
              src={VictoriaM}
              alt="Victoria Moore"
              className="avatar-image" />


            <p>Victoria is aspiring to be a software developer, she is especially looking forward to continuing to expand her knowledge and problem-solving skills that are needed for different areas of development. In her free time, she loves reading, watching films, and occasionally knitting and other crafts.</p>
          </div>

          <div className="item">
            <h2>Sejin</h2>

            <img
              src={SejinH}
              alt="Sejin Hwang"
              className="avatar-image" />


            <p>Sejin is a full stack web developer who enjoys the journey of every challenge. He is always looking to learn and improve, whether that through software development, sports, or even a simple youtube video.  You can find him often golfing, playing basketball, or enjoying the great outdoors of the PNW.   </p>
          </div>

          <div className="item">
            <h2>Zack</h2>

            <img
              src={ZackZ}
              alt="Zack Zboncak"
              className="avatar-image" />


            <p>Zack is a husband and father of three young boys. He has a wide array of interest, which shows from his degrees in physics and Old Testament. A problem solver at heart, he applies his newfound love for coding to projects like Get Meals. You can checkout his other projects <a href="https://zzboncak.github.io/Portfolio/" target="_blank" rel='noreferrer noopener'>here</a>.</p>
          </div>

          <div className="item">
            <h2>Peggy</h2>

            <img
              src={PeggyS}
              alt="avatar"
              className="avatar-image" />


            <p>Peggy is a Senior QA Engineer whose main goal for every project that she works on is being an advocate for the user. In her spare time she enjoys cooking plant based meals, running and seeing as much live music as possible.</p>
          </div>

        </div>
      </React.Fragment>




    );
  }
}

export default About;