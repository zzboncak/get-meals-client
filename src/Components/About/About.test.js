import React from 'react';
import ReactDOM from 'react-dom';
import About from './About';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';

describe(`About component`, () => {
  it('renders without crashing', () => {
      const div = document.createElement('div');
      ReactDOM.render(<About />, div);
      ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the component as expected', () => {
    const tree = renderer
      .create(
        <BrowserRouter>
            <About />
        </BrowserRouter>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
  });
})
