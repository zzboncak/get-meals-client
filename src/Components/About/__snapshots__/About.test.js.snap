// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component renders the component as expected 1`] = `
Array [
  <div
    className="about-header"
  >
    <h1>
      About Get Meals
    </h1>
    <p>
      Get Meals was created by a group of students at Thinkful who all of the like mind of wanting to create something that will help people. The app was born in the midst of the COVID-19 out of a want to help people easily find places where they can get free meals, the location of food banks and places that provide discounts to our military, healthcare workers, etc.
    </p>
    <h1>
      Meet the Team
    </h1>
  </div>,
  <div
    className="parentView"
  >
    <div
      className="item"
    >
      <h2>
        John
      </h2>
      <img
        alt="John's picture"
        className="avatar-image"
        src="JohnL.jpeg"
      />
      <p>
        John is a bioengineer transitioning to the software development world. He's just a nerd. When he has free time, John can be found doing something active outdoors (e.g. hiking a 14er, snowboarding). He is currently based out of Denver, CO. 
        <a
          href="https://johlee92.github.io/personalPortfolio/"
          rel="noreferrer noopener"
          target="_blank"
        >
          Here
        </a>
         is his portfolio.
      </p>
    </div>
    <div
      className="item"
    >
      <h2>
        Victoria
      </h2>
      <img
        alt="Victoria's picture"
        className="avatar-image"
        src="VictoriaM.jpeg"
      />
      <p>
        About text here.
      </p>
    </div>
    <div
      className="item"
    >
      <h2>
        Sejin
      </h2>
      <img
        alt="Sejin's picture"
        className="avatar-image"
        src="placeholder.jpg"
      />
      <p>
        About text here.
      </p>
    </div>
    <div
      className="item"
    >
      <h2>
        Zack
      </h2>
      <img
        alt="Zach's picture"
        className="avatar-image"
        src="ZackZ.jpeg"
      />
      <p>
        Zack is a husband and father of three young boys. He has a wide array of interest, which shows from his degrees in physics and Old Testament. A problem solver at heart, he applies his newfound love for coding to projects like Get Meals. You can checkout his other projects 
        <a
          href="https://zzboncak.github.io/Portfolio/"
          rel="noreferrer noopener"
          target="_blank"
        >
          here
        </a>
        .
      </p>
    </div>
    <div
      className="item"
    >
      <h2>
        Peggy
      </h2>
      <img
        alt="Peggy's picture"
        className="avatar-image"
        src="placeholder.jpg"
      />
      <p>
        About text here.
      </p>
    </div>
  </div>,
]
`;
